src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurSAEFileFixture.java

```diff
@@ -37,7 +37,8 @@
     DOCUMENT_NUMBER_TEST(4, 200.00),
     EMPLOYEE_NAME_TEST(4, 200.00),
     CASH_AND_CARD_TEST(2, 105.00),
-    CANCELED_TRIP_TEST(2, -432.40);
+    CANCELED_TRIP_TEST(2, -432.40),
+    PDP_TEST(4, 1000);

```

Comment:
The amount total is not lining up with the detail lines from the other fixture. However, since this enum constant is not actively used by the unit test code that needs the correct value, it may be okay to leave it as-is for now, and then fix it later if problems arise.